### ISSUE TITLE
Improve error message for bulk export in FHIR batches & transactions

### ISSUE THREAD

**Issue by CaitlinV39** [microsoft/fhir-server#1459]
**User story**
Currently, export requests in batches and transactions will return a 400 because the Accept and Prefer headers cannot be provided. The error message specifies: "Value supplied for the \"Accept\" header is not supported.".

Instead, return Not Supported and update the error message to explain that export operations are not supported in batches and transactions.

**Acceptance criteria**
If users try to put an export request in a batch or transaction, it will fail with an appropriate error message.

**Comment by mattkoch614** [comment]
Hi @CaitlinV39 - Can you provide an example of a request URI a user would make that results in this 400? 

Are you referring to export as defined here: http://hl7.org/fhir/us/bulkdata/2019May/export/index.html ?

**Comment by CaitlinV39** [comment]
@mattkoch614 Yes, that is the export we are looking at. Basically on this you can't invoke $export from within a bundle. This item is tracking to update the error message to make it a little clearer that you are doing an unsupported method.

**Comment by v-kulkarnipa** [comment]
Could Olympus team provide clear steps to reproduce it. Also we have noticed that there is one commit added in pull request (
https://github.com/microsoft/fhir-server/pull/2719), to resolve this issue. And this PR seems to be merged. Is this still something we need to do?

**Comment by evachen96** [comment]
@EXPEkesheth - We think this can be closed, as the PR fix was merged
